src/styles/index.scss

```diff
@@ -1,3 +1,127 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 400;
+  color: #060b35;
+}
+
+p {
+  margin: 0;
+}
+
+:root {
+  --main-accent: #060b35;
+  --secondary: #616070;
+  --blue-accent: #00ACDC;
+}
+
+.product {
+  margin: 50px;
+
+  &__box {
+    max-width: 200px;
+    padding: 32px 16px 16px;
+    box-sizing: border-box;
+    border-radius: 5px;
+    border: 1px solid #f3f3f3;
+    background: #fff;
+  }
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+    background: url('../images/imac.jpeg') center no-repeat;
+    background-size: cover;
+  }
+
+  &__description {
+    margin-bottom: 16px;
+  }
+
+  &__name {
+    font-weight: 500;
+    font-size: 12px;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    color: var(--secondary);
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-bottom: 24px;
+  }
+
+  &__review-count {
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__stars {
```

Comment:
not a mistake generally, but at least FYI:
you could just use `stars` block from the previous task (copy the file) - it's ok to use BEM block inside another one